go/web/skeleton/Makefile

```
@@ -40,34 +40,19 @@ p2p-image:
 p2p-build: lint service-build service-push ## Builds the service image and pushes it to the registry
 
 .PHONY: p2p-functional
-p2p-functional: namespace = $(tenant_name)-functional
+p2p-functional: namespace_environment=functional
```

Could we rename this variable?
```suggestion
p2p-functional: test_type=functional
```

---

there is a bit more to update if we want to rename it, I'll leave it as it is for now (as it is a minor change), but will keep it in mind for next story and update accordingly.